src/test/java/hudson/plugins/copyartifact/TriggeredBuildSelectorTest.java

```diff
@@ -46,11 +47,8 @@
 
 import org.junit.Rule;
 import org.junit.Test;
-import org.jvnet.hudson.test.Bug;
-import org.jvnet.hudson.test.ExtractResourceSCM;
-import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.*;
```

Comment:
You'd better to configure your IDE not to use wildcard imports. It's often considered a bad style.